cli/src/commands/rebase.rs

```diff
@@ -157,8 +159,32 @@ pub(crate) struct RebaseArgs {
     revision: Option<RevisionArg>,
     /// The revision(s) to rebase onto (can be repeated to create a merge
     /// commit)
-    #[arg(long, short, required = true)]
+    #[arg(long, short)]
     destination: Vec<RevisionArg>,
+    /// The revision(s) to insert after (can be repeated to create a merge
+    /// commit)
+    ///
+    /// Only works with `-r`.
+    #[arg(
+        long,
+        short = 'A',
+        visible_alias = "after",
+        conflicts_with = "source",
+        conflicts_with = "branch"
+    )]
+    insert_after: Vec<RevisionArg>,
+    /// The revision(s) to insert before (can be repeated to create a merge
+    /// commit)
```

Comment:
I suppose it doesn't necessarily create a merge (if all the revisions have a single shared parent), but it's probably fine to not mention that case.